python/ray/scripts/scripts.py

```diff
@@ -824,8 +819,8 @@ def rsync_up(cluster_config_file, source, target, cluster_name, all_nodes):
     type=str,
     help="(deprecated) Use '-- --arg1 --arg2' for script args.")
 @click.argument("script_args", nargs=-1)
-def submit(cluster_config_file, docker, screen, tmux, stop, start,
-           cluster_name, port_forward, script, args, script_args):
+def submit(cluster_config_file, screen, tmux, stop, start, cluster_name,
+           port_forward, script, args, script_args):
     """Uploads and runs a script on the specified cluster.
```

Comment:
Are we not doing 'run env' for submit too?

Reply:
It isn't necessary and leads to an easy way for users to shoot themselves in the foot because the host doesn't necessarily have the same configuration as the container. If a user really wanted to run a python script from outside the container, they could rsync it and then exec it.